ERRORS.md

```diff
@@ -0,0 +1,59 @@
+# Error Propagation in Broccoli & Ember CLI
+
+## Quick Summary
+
+This is a short summary of how errors during build time are surfaced to the browser. Let's look at a particular example.
+
+Let's say you have the following template:
+
+```handlebars
+<p>Sup you guise<p>
+```
+
+There's an unclosd `p` tag. Seems bad.
+
+Here's what is going to happen:
+
++ `ember-template-compiler` throws an exception
+  + `location`
+  + `message`
+  + `stack`
+
+Error contains information of where in the file error occured and what kind of error it was.
+
++ `ember-cli-htmlbars` will catch the error from `ember-template-compiler`.
+
+Because `ember-cli-htmlbars` is based on `broccoli-persistent-filter`, technically `broccoli-persistent-filter`
+will catch the exception first. For example, we attach `file` and `treeDir` information to errors [here](https://github.com/stefanpenner/broccoli-persistent-filter/blob/master/index.js#L283-L288).
```

Comment:
Link should be to a tag (so I doesn't drift and point to the wrong code as changes are made upstream)...